content/operate/rc/resilient-apps.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: Develop highly available and resilient apps with Redis Cloud 
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: 
+hideListLinks: true
+linkTitle: Develop highly available apps
+weight: 50
+---
+
+You can set up your Redis Cloud databases and Redis Client libraries that can help your app re-connect to your database after unexpected failover events or network outages and minimize data losses. 
+
+## Set up databases
+
+These settings can be turned on when you create or edit your database. 
+
+### Enable data persistence
+
+[Data persistence]({{< relref "/operate/rc/databases/configuration/high-availability" >}})
```

Comment:
The link by itself looks a bit odd. Maybe put the link around "data persistence" in the line below or add some explanation here?

Reply:
Oops, I think I started that and then didn't finish the sentence. Thanks for the catch.